server/public/model/remote_cluster.go

```diff
@@ -437,9 +446,28 @@ func (rci *RemoteClusterInvite) Decrypt(encrypted []byte, password string) error
 	salt := encrypted[:16]
 	encrypted = encrypted[16:]
 
-	key, err := scrypt.Key([]byte(password), salt, 32768, 8, 1, 32)
-	if err != nil {
-		return err
+	// Try PBKDF2 first (for version 3+)
+	if err := rci.tryDecrypt(encrypted, password, salt, true); err == nil {
+		return nil
+	}
+
+	// Fall back to scrypt (for older versions)
+	return rci.tryDecrypt(encrypted, password, salt, false)
+}
+
+func (rci *RemoteClusterInvite) tryDecrypt(encrypted []byte, password string, salt []byte, usePBKDF2 bool) error {
+	var key []byte
+	var err error
+
+	if usePBKDF2 {
+		// Use PBKDF2 for version 3 and above
+		key = pbkdf2.Key([]byte(password), salt, 100000, 32, sha256.New)
```

Comment:
How was the 100k value chosen? Looking around, some(AI) say to "increase PBKDF2 iterations to 600,000 minimum for OWASP compliance (2024)". it is also apparently "3x below NIST draft standards (310,000)" (not sure we are targeting these necessarily). 0/5 also to move the value to some named constant.